### DISCUSSION TITLE
Problème d'affichage des heures - calendrier

### DISCUSSION THREAD

**Discussion by Rouqueen** [JeromeDevome/GRR#376] [Q&A]
Bonjour
Nous avons installé la version 4.3.2 de GRR pour notre laboratoire de recherche.  Cet outil semble être très prometteur pour nos besoins. On le test avant grand déploiement pour nos usagers et nous avons le problème suivant.  Les plages disponibles pour les usagers sont disponibles 7h plus tard que l'heure actuelle.  Nous offrons la possibilité aux usagers de réserver sur une plage de 7h00 à 19h00.  Le calendrier montre bien cette plage horaire.  Disons qu'il est 11h du matin et l'usager ouvre le portait de réservation, il n'a pas accès au plages avant 18h (pas de petit plus sur la case horaire).
<img width="1422" alt="grr" src="https://github.com/JeromeDevome/GRR/assets/118198003/630911a4-a92b-4c6b-92c8-3bddf52cb323">
<img width="1422" alt="grr" src="https://github.com/JeromeDevome/GRR/assets/118198003/630911a4-a92b-4c6b-92c8-3bddf52cb323">

On peut voir que l'heure est correcte dans le haut de la page à droite.  Mais que l'usager ne peut pas réserver à partir de 11h mais seulement plus tard.

Je suis allée valider le document config.inc.php et le time zone était réglé à Europe/Paris.  Je l'ai mis à Americas/New_York (je suis au canada, nous avons le même time zone que new york où nous sommes).  Mais ça ne change rien au problème.

Je ne sais pas trop où chercher pour tenter de régler ce problème car actuellement, l'outil ne sera pas utilisable si on ne peut pas trouver et régler la source du problème. GRR répond en tout point à nos besoins et nous aimerions vraiment pouvoir l'utiliser.

Je ne suis pas familière avec le php et la personne qui s'occupait d'initier le projet n'est plus là.  Alors, toute aide est appréciée!
Caroline, technicienne électronique.

**Comment by ynaessens** [comment]
Bonjour,
il faudrait vérifier que l'heure affichée est bien celle du serveur.
Si ce n'est pas l'origine du décalage, peut-être essayer de voir si la version 3.5.1d gère mieux ce problème de fuseau horaire.
Merci de nous rendre compte de vos succès/échecs.
Cordialement,
YN
